app/src/main/java/org/simple/clinic/editpatient/EditPatientScreen.kt

```diff
@@ -480,7 +480,7 @@ class EditPatientScreen(context: Context, attributeSet: AttributeSet) : Relative
         ColonyOrVillageEmpty -> showEmptyColonyOrVillageError(true)
         DistrictEmpty -> showEmptyDistrictError(true)
         StateEmpty -> showEmptyStateError(true)
-        BothDateOfBirthAndAgeAdsent -> showAgeEmptyError(true)
+        BothDateOfBirthAndAgeAbsent -> showAgeEmptyError(true)
```

Comment:
Since it's not related to this PR, can we not do this here? Please open a different PR for this.